storage/src/mmr/bitmap.rs

```diff
@@ -395,25 +372,41 @@ impl<H: CHasher, const N: usize> Bitmap<H, N> {
         self.mmr.update_leaf(hasher, leaf_pos, chunk).await
     }
 
-    /// Return the root of the Merkle tree over the bitmap.
+    /// Return the root digest against which inclusion proofs can be verified.
     ///
-    /// # Warning
+    /// ## Format
```

Comment:
nit: in other places, we just do `# Format`

Reply:
done